src/lib/rmt/utils.rb

```diff
@@ -62,7 +62,7 @@ def write_config_file(config)
 
     # Runs a command and returns the exit code
     def run_command(command, *params)
-      params = params.map { |p| Yast::String.Quote(p) }
+      params = params.map { |p| Shellwords.escape(p) }
```

Comment:
I'd suggest adding an explicit `require 'shellwords'` -- it seems it is required implicitly (probably by Yast).

Reply:
Ok, I pushed the change, but I have to test it more